### ISSUE TITLE
Paracentral hyper-autofluorecence

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7338]
Is this a synonym of [Perifoveal ring of hyperautofluorescence HP:0030629](https://hpo.jax.org/app/browse/term/HP:0030629)?

PMID:33891002
9/16 patients  OMIM:619649

**Comment by pnrobinson** [comment]
@Sukramg
could you provide wisdom for this?

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000769
- label: Paracentral hyper-autofluorecence
- parents: [Abnormal fundus autofluorescence imaging (HP:0030602)](https://hpo.jax.org/app/browse/term/HP:0030602)
- pmid: 
- def: Increased autofluorescence in areas adjacent to the central macula (but not including the very center).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7338
- synonyms:
